doc/manuals/user/update_operators.md

```diff
@@ -238,12 +238,114 @@ For instance, if the preexisting value of attribute A in entity E is `[1, 2, 3]`
 POST /v2/entities/E/attrs/A
 {
   "value": { "$pullAll": [2, 3] },
-  "type": "Number"
+  "type": "Array"
 }
 ```
 
 would change the value of attribute A to `[1]`.
 
+### `$set`
+
+To be used with attributes which value is an object to add/update a sub-key in the
+object without modifying any other sub-keys.
+
+For instance, if the preexisting value of attribute A in entity E is `{"X": 1, "Y": 2}` the
+following request:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": { "$set": {"Y": 20, "Z": 30} },
+  "type": "Object"
+}
+```
+
+would change the value of attribute A to `{"X": 1, "Y": 20, "Z": 30}`.
+
```

Comment:
Reading this, I have a doubt, what if executes the following request?

```
 POST /v2/entities/E/attrs/A
 {
   "value": { "$set": {"X": 10} },
   "type": "Object"
 }
```

Reply:
Clarified at skype. NTC